interface/wx/grid.h

```diff
@@ -4425,6 +4459,15 @@ class wxGrid : public wxScrolledWindow
     wxArrayInt CalcColLabelsExposed( const wxRegion& reg );
     wxGridCellCoordsArray CalcCellsExposed( const wxRegion& reg );
 
+    /**
+        Split a string containing newline characters into an array of
+        strings and return the number of lines
+    */
+    void StringToLines( const wxString& value, wxArrayString& lines ) const;
```

Comment:
This is confusing, it doesn't return the number of lines, does it?
It should probably not be in the public API at all. There is `wxSplit()` for the same purpose apparently.